packages/react/counter/src/__test__/mswHandlers.ts

```diff
@@ -1,11 +1,10 @@
-import { graphql } from 'msw'
-
 import { server } from '@/testing/mocks/server'
 
 import { counter } from './mockData'
+import { mockCounterQuery } from './mockHandlers'
 
 const handlers = [
-  graphql.query('Counter', (req, res, ctx) => {
+  mockCounterQuery((req, res, ctx) => {
```

Comment:
I believe we should be using the GraphQL code generator provided mock handlers. They give the best type safety.